### PR TITLE
Update aws_queue_worker.rb

### PR DESCRIPTION
**Pull request by Sjeanpierre**
Fixes issue where ApproximateReceiveCount was not part of payload from sqs

### CODE REVIEW

**Comment by chrisbarber86** [comment]
👍

**Comment by chilts1978** [comment]
- 1

File: eventq_aws/lib/eventq_aws/aws_queue_worker.rb [hunk 1]
```diff
@@ -57,14 +57,14 @@ def start(queue, options = {}, &block)
                                                           queue_url: q,
                                                           max_number_of_messages: 1,
                                                           wait_time_seconds: 1,
-                                                          message_attribute_names: ['ApproximateReceiveCount']
+                                                          attribute_names: ['ApproximateReceiveCount']
```
Comment (vaughanbrittonsage) [inline eventq_aws/lib/eventq_aws/aws_queue_worker.rb]: According to the AWS SQS documentation here: http://docs.aws.amazon.com/sdkforruby/api/Aws/SQS/Client.html this is not the correct way to request attribute names:

`resp = client.receive_message({
  queue_url: "String", # required
  attribute_names: ["Policy"], # accepts Policy, VisibilityTimeout, MaximumMessageSize, MessageRetentionPeriod, ApproximateNumberOfMessages, ApproximateNumberOfMessagesNotVisible, CreatedTimestamp, LastModifiedTimestamp, QueueArn, ApproximateNumberOfMessagesDelayed, DelaySeconds, ReceiveMessageWaitTimeSeconds, RedrivePolicy
  message_attribute_names: ["MessageAttributeName"],
  max_number_of_messages: 1,
  visibility_timeout: 1,
  wait_time_seconds: 1,
})`

The previous way is the correct way, curious as to what drove this change?

File: eventq_aws/lib/eventq_aws/aws_queue_worker.rb [hunk 2]
```diff
@@ -57,14 +57,14 @@ def start(queue, options = {}, &block)
...
                                                       })
 
                 #check that a message was received
                 if response.messages.length > 0
 
                   msg = response.messages[0]
-                  retry_attempts = msg.message_attributes['ApproximateReceiveCount'].to_i
+                  retry_attempts = msg.attributes['ApproximateReceiveCount'].to_i
```
Comment (vaughanbrittonsage) [inline eventq_aws/lib/eventq_aws/aws_queue_worker.rb:67]: 👍 This is correct according to the AWS documentation. Must have missed this, thought I had ran a test for this previously.

**Comment by vaughanbrittonsage** [comment]
I have made these changes plus some additional changes to a separate branch and tested. I will release the other branch for these changes. Thanks

**Pull request closed without merging**